tests/test_matcher.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+from pytest_httpserver import HTTPServer
+
+
+def test_expect_methods_are_equivalent(httpserver: HTTPServer):
```

Comment:
in the spite of my below comment, this would need to be renamed:

```suggestion
def test_expect_method(httpserver: HTTPServer):
```